.github/workflows/graalwasm-micronaut-excelice.yml

```diff
@@ -0,0 +1,48 @@
+name: Test GraalWasm Micronaut Excelice Demo
+
+on:
+  push:
+    paths:
+      - 'graalwasm/graalwasm-micronaut-excelice/**'
+      - '.github/workflows/graalwasm-micronaut-excelice.yml'
+  pull_request:
+    paths:
+      - 'graalwasm/graalwasm-micronaut-excelice/**'
+      - '.github/workflows/graalwasm-micronaut-excelice.yml'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  run:
+    name: 'graalwasm-micronaut-excelice'
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          java-version: '21'
+          distribution: 'graalvm'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          cache: 'maven'
+          native-image-job-reports: 'true'
+
+      - name: Install Node.js & Wasm for GraalVM
+        run: |
+          gu install js wasm nodejs
```

Comment:
This doesn't work because `gu` is longer available in GraalVM for JDK21+

```suggestion
```

The demo depends on JS and Wasm, and the dependencies are pulled in correctly via the _pom.xml_.